source/_components/media_player.epson.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: "Epson"
+description: "Instructions on how to integrate Epson projector into Home Assistant."
+date: 2018-03-24 14:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: epson.png
+ha_category: Media Player
+featured: true
+ha_release: "0.66"
+ha_iot_class: "Local Push"
+---
+
+
+The `epson` platform allows you to control a Epson projector from Home Assistant.
+
+To add Epson to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: epson
+    host: 192.168.0.123
+```
+
+Configuration variables:
```

Comment:
Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration